src/screens/AboutScreen.js

```diff
@@ -55,6 +57,8 @@ export const AboutScreen = ({ navigation }) => {
   const { sections = {} } = globalSettings;
   const { headlineAbout = texts.homeTitles.about } = sections;
 
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.MORE);
```

Comment:
this could also be before the other logic

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/109/commits/f22a1566b20d2eded43188bff5bc48e012167c41